anim.js

```diff
@@ -87,13 +93,13 @@ class Anim {
   runLoop(universe) {
     const doAnimation = () => {
       this.run(universe, () => {
-        setImmediate(doAnimation)
-      })
-    }
+        setImmediate(doAnimation);
+      });
+    };
 
-    doAnimation()
+    doAnimation();
 
-    return this
+    return this;
```

Comment:
I think enforcing a line-ending style (so that we always have a semicolon and not randomly) makes sense.
But also: Could you add an lint rule for it, so we have this enforced for further changes?